packages/application-components/src/components/page-content-containers/page-content-wide/page-content-wide.tsx

```diff
@@ -0,0 +1,80 @@
+import { Children, ReactNode } from 'react';
+import styled from '@emotion/styled';
+
+export type TPageContentWide = {
+  columns: '1' | '1/1' | '2/1';
+  gapSize: '10' | '20';
+  children: ReactNode;
+};
+
+const LeftContentMarginColumn = styled.div`
+  grid-area: left-margin;
+`;
+
+const MainContentColumn = styled.div<
+  Pick<TPageContentWide, 'columns' | 'gapSize' | 'children'>
+>`
+  grid-area: main-content;
+  display: grid;
+  grid-template-areas: ${(props) =>
+    props.columns === '1' ? 'none' : '"left-column right-column"'};
+  grid-template-columns: ${(props) => {
+    switch (props.columns) {
+      case '1/1':
+        return '1fr 1fr';
+      case '2/1':
+        return '2fr 1fr';
+      default:
+        return '1fr';
+    }
+  }};
+  gap: ${(props) => (props.gapSize === '10' ? '32px' : '64px')};
```

Comment:
Should we use design tokens here?

```suggestion
  gap: ${(props) => (props.gapSize === '10' ? designTokens.spacing50 : designTokens.spacing70)};
```

Reply:
Totally.

Reply:
Updated in this commit: 5e306e1c1fabf0f61996604f1fb07d166f41e12e